example/widget_surveyor.dart

```diff
@@ -146,16 +152,14 @@ class AnalysisResult {
         widgetReferences = {} {
     var map = json['widgets'];
     for (var entry in map.entries) {
-      widgetReferences[entry.key] = entry.value as List<String>;
+      widgetReferences[entry.key] = List.from(entry.value);
```

Comment:
I'm not sure which is better from either a readability or performance perspective, but an alternative that's slightly shorter to write is `entry.value.toList()`. I'd leave it as is unless you really like the alternate better.

Reply:
Not sure either to be honest.  Still puzzling over it...